php-practice/section6.php

```diff
@@ -164,9 +191,18 @@ function getTriangleNumber($x) {
  * @return int
  */
 function sum1To100Evens() {
-  // YOUR CODE HERE
+  $value = 0;
+
+    for($i = 0; $i < 100; $i+=2) {
+      $value += $i;
+    }
+
+    return $value;
 }
 
+echo sum1To100Evens().PHP_EOL;
```

Comment:
if ($i % 2 == 0)

I don't know if my code says what I'm thinking so I'm also going to explain what I'm thinking.

I put $i % 2 == 0 because we want to divide the number we are increasing by an increment of 2 ($i) by 2 checking to see if it equals a remainder of 0 because an even number divided by 2 should return a remainder of 0. I'm just not sure if I put the right code for that syntax